runhouse/cli_utils.py

```diff
@@ -0,0 +1,522 @@
+                        # so we could calculate function's running time.
+                        func_end_time_utc = datetime.fromtimestamp(
+                            time.time(), tz=timezone.utc
+                        )
+
+                        func_running_time = (
+                            func_end_time_utc - func_start_time_utc
+                        ).total_seconds()
+
+                        is_func_running: str = f" [italic bright_green]Running for {func_running_time} seconds[/italic bright_green]"
+
+                    elif (
+                        resource_type == "runhouse.Function"
+                        and not active_function_calls
+                    ):
+                        is_func_running: str = " [italic bright_yellow]Currently not running[/italic bright_yellow]"
+
+                    else:
+                        is_func_running: str = ""
+
+                    resource_info_str = resource_info_str + is_func_running
+
+                    console.print(resource_info_str)
+
+
+def print_cloud_properties(cluster_config: dict):
+    from runhouse.main import console
+
+    cloud_properties = cluster_config.get("launched_properties", None)
+    if not cloud_properties:
+        return
+    cloud = cloud_properties.get("cloud")
+    instance_type = cloud_properties.get("instance_type")
+    region = cloud_properties.get("region")
+    cost_per_hour = cloud_properties.get("cost_per_hour")
+
+    has_cuda = cluster_config.get("has_cuda", False)
+    cost_emoji = "💰" if has_cuda else "💸"
+
+    num_of_cpus = cloud_properties.get("num_cpus") or len(cluster_config.get("ips"))
+    num_of_gpus = 0
+    cluster_accelerators = cloud_properties.get("accelerators", None)
+    gpu_types = set()
+    if cluster_accelerators:
+        for k, v in cluster_accelerators.items():
+            num_of_gpus = num_of_gpus + int(v)
+            gpu_types.add(k)
+
+    console.print(
+        f"[reset]🤖 {cloud} {instance_type} cluster | 🌍 {region} | {cost_emoji} ${cost_per_hour}/hr"
+    )
+    cpus_gpus_info_str = f"CPUs: {int(float(num_of_cpus))}"
+    if num_of_gpus > 0:
+        gpu_types_str = gpu_types.pop()
+        for gpu_type in gpu_types:
+            gpu_types_str = gpu_types_str + f", {gpu_type}"
+        cpus_gpus_info_str = (
+            cpus_gpus_info_str + f" | GPUs: {num_of_gpus} (Type(s): {gpu_types_str})"
+        )
+    console.print(f"[reset]{cpus_gpus_info_str}")
+
+
+def print_status(status_data: dict, current_cluster) -> None:
+    from runhouse.main import console
+
+    """Prints the status of the cluster to the console"""
+    cluster_config = status_data.get("cluster_config")
+    env_servlet_processes = status_data.get("env_servlet_processes")
+
+    cluster_name = cluster_config.get("name", None)
+
+    if cluster_name:
+        api_server_url = cluster_config.get(
+            "api_server_url", rh.configs.get("api_server_url")
+        )
+        api_server_url = api_server_url.replace(
```

Comment:
what are we doing with the URL here? we could just hard code the link, it's not likely to change

Reply:
we convert the api server url to the runhouse UI link. We could hard code it, but than it will be the same for den prod and den dev. But i guess it is ok, since den-dev is used just in house.